🙏  add GDPR delete to the FAQ (#232)
configcat/docs: website/docs/faq.md

- All tags
- All segments

> ### How long do you keep my data?

Reviewer: ```suggestion
### How long does ConfigCat keep my data?
```

We keep organization data as long as we see activity in that organization.
After several months of inactivity, we send you a series of email notifications about or plans to delete your organization and all associated data.
We consider an organization inactive if it meets all of the following criteria:
- no audit log events are generated in the organization,

> - no valid REST API calls are made to the organization,

Reviewer: ez melyik API? Rest apirol mashol nem irunk. van dashboard api es public mgmt api.

Author: beírtam most, hogy Public Management API

> - no valid REST API calls are made to the organization,

Author: ```suggestion
- no valid calls are made to the organization via the Public Management API,
```

> - none of the organization's config JSONs are downloaded,

Reviewer: ```suggestion
- config JSONs aren't downloaded from the ConfigCat CDN,
```
